### ISSUE TITLE
Doesn't work on 64 bit devices

### ISSUE THREAD

**Issue by ChuckPinkert** [ChuckPinkert/TTTAttributedLabel#1]
When TTTAttributedLabel is used on a  64 bit device the following exception happens:

```
Wrapper type 'AttributedLabel.TTTAttributedLabel' is missing its native ObjectiveC class 'TTTAttributedLabel'.
```

**Comment by bholmes** [comment]
Try adding 

LinkerFlags = "-ObjC", to your linkwith attribute.  Also added LinkTarget.Arm64 as well.

[assembly: LinkWith ("TTTAttributedLabelSDK.a", 
    LinkTarget.Simulator | LinkTarget.ArmV7 | LinkTarget.Arm64, 
    LinkerFlags = "-ObjC",
    SmartLink = true, 
    ForceLoad = true)]

**Comment by ChuckPinkert** [comment]
Thanks @bholmes! That did it

This commit fixes it: a012d253ed594ea6a12
